docs/assemblies/proc_hotfixing-the-data-plane-container-content-images.adoc

```diff
@@ -0,0 +1,30 @@
+[id="proc_hotfixing-the-data-plane-container-content-images-{context}"]
+= Hotfixing the data plane container content with images
+
+[role="_abstract"]
+
+When container hotfix content is delivered as images, the container processes need to be restarted to use the new images. This will be accomplished by creating a new `OpenStackDataPlaneDeployment`.
+
+.Procedure
+
+. Optional: Prepare the container hotfix image in a container registry where the image can be pulled by affected data plane nodes
++
+----
+$ podman pull <container_registry>/<container_project>/<container_image>:<container_tag>
+$ podman tag <container_registry>/<container_project>/<container_image>:<container_tag> <updated_container_registry>/<updated_container_project>/<container_image>:<container_tag>
+$ podman push <updated_container_registry>/<updated_container_project>/<container_image>:<container_tag>
+----
++
+* Replace `<container_registry>` with the source registry for the already hotfixed container image
+* Replace `<container_project>` with the source project for the already hotfixed container image
+* Replace `<container_image>` with the already hotfixed container image
+* Replace `<container_tag>` with the tag being for the already hotfixed container image
+* Replace `<updated_container_registry>` with a container registry to serve the hotfixed container image. The OCP internal container image registry can be used.
+* Replace `<updated_container_project>` with a container project to use for the hotfixed container image.
+
```

Comment:
it didn't render as bullet points:
![image](https://github.com/openstack-k8s-operators/dataplane-operator/assets/17153022/13ec0c85-b7fc-4156-977c-f55a5b997014)
https://logserver.rdoproject.org/01/701/934bc9f064dc2630319cb4fc6704eff5abf367dd/github-check/dataplane-operator-docs-preview/9371f30/docs_build/docs_build/dataplane/index-upstream.html#proc_hotfixing-the-data-plane-container-content-rpms-dataplane

Reply:
I wonder if it is a + missing earlier - can you see if there is a + after the line 18 step instruction:

"From a RHEL workstation, server, or virtual machine, collect the hotfix RPMs into a new directory"
+
----

Reply:
this was resolved